uint16 HeaderInfo
uint16 RotationalInfo
velodyne/LaserDataMsg[32] LaserData
